examples/40_paper/2018_ida_strang_example.py

```diff
@@ -9,9 +9,95 @@
 ~~~~~~~~~~~
 
 | Don't Rule Out Simple Models Prematurely: A Large Scale Benchmark Comparing Linear and Non-linear Classifiers in OpenML
-| Benjamin Strang, Pieter Putten, Jan van Rijn and Frank Hutter
+| Benjamin Strang, Peter van der Putten, Jan N. van Rijn and Frank Hutter
 | In *Advances in Intelligent Data Analysis XVII 17th International Symposium*, 2018
 | Available at https://link.springer.com/chapter/10.1007%2F978-3-030-01768-2_25
-
-This is currently a placeholder.
 """
+import matplotlib.pyplot as plt
+import openml
+import pandas as pd
+
+##############################################################################
+# A basic step for each data-mining or machine learning task is to determine
+# which model to choose based on the problem and the data at hand. In this
+# work we investigate when non-linear classifiers outperform linear
+# classifiers by means of a large scale experiment.
+#
+# The paper is accompanied with a study object, containing all relevant tasks
+# and runs (study id = 123). The paper features three experiment classes:
+# Support Vector Machines (SVM), Neural Networks (NN) and Decision Trees (DT).
+# This example demonstrates how to reproduce the plots, comparing two
+# classifiers given the OpenML flow ids. Note that this allows us to reproduce
+# the SVM and NN experiment, but not the DT experiment, as this requires a bit
+# more effort to distinguish the same flow with different hyperparameter
+# values.
+##############################################################################
```

Comment:
Could you please remove this line of hashes? It renders the first line of the "block" as a header (which it clearly isn't) and adds an ugly horizontal bar into the output.